config/sidekiq.yml

```diff
@@ -12,3 +12,7 @@
   - bulk_republishing
   - sync_checks
   - asset_migration
+:schedule:
+  check_editions_links:
```

Comment:
Can we name this as close as possible to the worker name but without the `Worker` suffix?